### ISSUE TITLE
Organisational IDs

### ISSUE THREAD

**Issue by trickvi** [openspending/fiscal-data-package#13]
According to a mailing list thread at IATI they are dropping support for Organisational IDs. This creates a problem for us and we need to find a replacement.

Suggestions:
- Annelise Parr says they will bring Organisational IDs back in version 2
- Tom Lee asks if Open Civic IDs can work

**Comment by timgdavies** [comment]
I'm interested in the approach the Fiscal Data Package is planning here.

The [Joined Up Data Alliance](https://twitter.com/joinedup_data) have been discussing Organisation IDs recently. The approach in [IATI](http://www.iatistandard.org) and [OCDS](http://standard.open-contracting.org) is broadly to develop a codelist of 'registration agencies' for organisation identifiers, and then encourage re-use of existing identifiers from those agencies.

This approach works well for companies, charities and other registered bodies, but not so well for government entities, where there is often not a robust list of entities (with the exception perhaps of the list of counter-parties kept in government accounting systems). 

One option may be to require publication of the list of counter parties along with names and address details as an accompanying part of publishing fiscal data.

**Comment by rgrp** [comment]
@timgdavies big +1 on point of organization ids. I do not think there is any settled view here at the moment and input and suggestions would great - in particular, [entities section of the spec](http://fiscal.dataprotocols.org/spec/#entities) does not currently specify any particular vocabulary for org identifiers.

Re public bodies (e.g. govs and their departments) there is http://publicbodies.org/ but it is still quite limited.

So, in summary: input warmly welcome.

**Comment by danfowler** [comment]
Trying to move this forward.

If we can agree that a single organization identifier scheme for all countries is not necessary as @jpmckinney suggests, perhaps what we can do as a first step is harmonize with the approach we've taken for the `location` dimension and specify a `codeList` (or `codelist`) pointing to one of:

- "public-bodies" (http://publicbodies.org/)
- "iati" (http://iatistandard.org/201/organisation-identifiers/)

This list can be expanded as a go forward.  What do you think @rgrp @jpmckinney @timgdavies ?

**Comment by timgdavies** [comment]
I would support this. 

The ideal short-term approach might be to ask IATI to add a code for PublicBodies.org to their [Organisation Registration Agency](http://iatistandard.org/201/codelists/OrganisationRegistrationAgency/) codelist, and then this actually collapses into the one approach.

In the long-term, we need to get that Organisation Registration Agency codelist adopted as a shared list by everyone involved in the Joined Up Data Alliance or some other suitable body that can co-ordinate between the standards setters.

**Comment by timgdavies** [comment]
(For reference, in Open Contracting we have four elements in our ```Identifier``` block for these situations:

* ```scheme``` - which should be a value from a codelist, such as the Organisation Registration Agency codelist;
* ```identifier``` - the identifier taken from that scheme (e.g. If scheme = GB-COH for UK Companies House, then identifier is the company number)
* ```legalName``` - for the full legal name (if known) of the entity;
* ```uri``` - for an optional URI to a register entry for the entity

You may want to consider whether ```scheme``` works as a property name for your use also.)

**Comment by jpmckinney** [comment]
+1 `scheme`

**Comment by pwalsh** [comment]
Moving to https://github.com/frictionlessdata/datapackage-fiscal/issues/8
